src/base/bootstrap-overrides/_base.scss

```diff
@@ -589,6 +589,24 @@ input[type=button], input[type=reset], input[type=submit] {
 	width: auto;
 }
 
+/*
+ * Explicit labels for checkbox and radio button inputs
+ *
+ * Allow the use of horizontal explicit labels instead of the default implicit
+ * labelling pattern for checkbox and radio button inputs provided by Bootstrap.
+ */
+.form-inline {
+	.label-inline {
+		font-weight: 400;
```

Comment:
There might be a variable available for this one.

Reply:
I've checked all the scss files to find a variable that set a font-weight to normal but nothing seems to fit the needs.